### PR TITLE
Google workspace description update in CreateUI

### PR DESCRIPTION
**Pull request by v-prasadboke**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-prasadboke** [comment]
image

update in data connector descripttion

**Comment by v-atulyadav** [comment]
Looking into this.

File: Solutions/GoogleWorkspaceReports/Data Connectors/GWorkspaceReports_API_FunctionApp.json [hunk 1]
```diff
@@ -163,14 +163,14 @@
   "instructionSteps": [
     {
       "title": "",
-      "description": ">**NOTE:** This connector uses Azure Functions to connect to the Google Reports API to pull its logs into Microsoft Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
+      "description": "**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias GWorkspaceReports and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/GoogleWorkspaceReports/Parsers/GWorkspaceActivityReports), on the second line of the query, enter the hostname(s) of your GWorkspaceReports device(s) and any other unique identifiers for the logstream. The function usually takes 10-15 minutes to activate after solution installation/update."
```
Comment (v-vdixit) [inline Solutions/GoogleWorkspaceReports/Data Connectors/GWorkspaceReports_API_FunctionApp.json:166]: @v-prasadboke, this change is not expected here, please ensure the required instructions stay intact, thanks

File: Solutions/GoogleWorkspaceReports/Data Connectors/GWorkspaceReports_API_FunctionApp.json [hunk 2]
```diff
@@ -163,14 +163,14 @@
...
     },
     {
       "title": "",
       "description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
     },
     {
-      "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-GWorkspaceReportsAPI-parser) to create the Kusto functions alias, **GWorkspaceActivityReports**"
+      "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias Google Workspace Reports and load the function code. The function usually takes 10-15 minutes to activate after solution installation/update."
```
Comment (v-vdixit) [inline Solutions/GoogleWorkspaceReports/Data Connectors/GWorkspaceReports_API_FunctionApp.json:173]: Please use updated text for this note here as follows: - **NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected which is deployed as part of the solution. To view the function code in Log Analytics, open Log Analytics/Microsoft Sentinel Logs blade, click Functions and search for the alias GWorkspaceReports and load the function code or click [here](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/GoogleWorkspaceReports/Parsers/GWorkspaceActivityReports), on the second line of the query, enter the hostname(s) of your GWorkspaceReports device(s) and any other unique identifiers for the logstream. The function usually takes 10-15 minutes to activate after solution installation/update.

File: Solutions/GoogleWorkspaceReports/Package/createUiDefinition.json
Comment (v-vdixit) [inline Solutions/GoogleWorkspaceReports/Package/createUiDefinition.json:1]: Could you please discard changes in this file, as they are mostly related to white spaces, thanks

**Review by v-vdixit — APPROVED** [review]
Looks good!

**Pull request merged**